components/layout/layout-root.tsx

```diff
@@ -3,11 +3,13 @@ import { GlobalHeader } from "./global-header";
 import { Navigation } from "./navigation";
 
 import { ScrollArrow } from "~/components/ui/scroll-arrow";
+import { FooterDiscussionSection } from "~/components/ui/footer-discussion-section";
 
 export const LayoutRoot: React.FC = ({ children }) => (
   <main className="flex flex-col w-full min-h-screen bg-gray-100">
     <GlobalHeader />
     {children}
+    <FooterDiscussionSection />
```

Comment:
Don't put this here. Put this at the bottom of `Page` instead.

The `Navigation` component below it is fixed, and will obscure is section.

![image](https://user-images.githubusercontent.com/5663877/128636708-91025151-53e9-4b1d-a5af-e45f07b52157.png)

The `Page` component has bottom padding to help alleviate this.

Reply:
Thank you, Mas. I've fixed it.

But some of the text is still blocked by the scroll-arrow. Is it necessary to add more padding right in the mobile view?
Or do you have any other suggestions?

<img src="https://user-images.githubusercontent.com/13805020/128637854-58be356c-a7f2-4533-95ba-fec99ecca77d.png" width="400">

Reply:
We can add extra padding on the `Page` component. We'll do that on a separate PR.

Reply:
Or if you want, you can add extra padding in the `FooterDiscussionSection` to give room at the bottom. @nasrulgunawan

Reply:
How about I add a footer like this, Mas @resir014?

<img src="https://user-images.githubusercontent.com/13805020/128707719-f1945503-33cd-408b-a1e8-2b916815e8e7.png" width="400" style="max-width:100%;">

Reply:
No. Let's not add any more than what's necessary.

Reply:
I learned a lot here. Thanks, Mas @resir014.

Okay, I try to add extra padding.
I added extra padding top and bottom (.py-8). Because if I add only the padding-bottom, it feels disproportionate.
Is it okay?

<img src="https://user-images.githubusercontent.com/13805020/128709092-bce85c06-7d89-4e45-a264-75ac557b5ecc.png" width="400" style="max-width:100%;">

Reply:
`pt-8 pb-16` is okay. @nasrulgunawan

Reply:
Okay Mas @resir014, I've fixed it.